### ISSUE TITLE
Composer install failed with new version of Composer

### ISSUE THREAD

**Issue by vbridier** [codezero-be/composer-preload-files#2]
Hello,

I use codezero/laravel-localized-routes package which depends on this one

Composer install error : 
```php
In Filesystem.php line 554:

  [TypeError]
  Composer\Util\Filesystem::normalizePath(): Argument #1 ($path) must be of type string, null given, called in /app/srcs/vendor/codezero/composer-preload-files/src/AutoloadGenerator.php on line 154


Exception trace:
  at phar:///usr/local/bin/composer/src/Composer/Util/Filesystem.php:554
 Composer\Util\Filesystem->normalizePath() at /app/srcs/vendor/codezero/composer-preload-files/src/AutoloadGenerator.php:154
 CodeZero\ComposerPreloadFiles\AutoloadGenerator->parsePreloadFiles() at /app/srcs/vendor/codezero/composer-preload-files/src/AutoloadGenerator.php:26
 CodeZero\ComposerPreloadFiles\AutoloadGenerator->addPreloadFilesToAutoloadFiles() at /app/srcs/vendor/codezero/composer-preload-files/src/Plugin.php:64
 CodeZero\ComposerPreloadFiles\Plugin->addPreloadFilesToAutoloadFiles() at phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:206
 Composer\EventDispatcher\EventDispatcher->doDispatch() at phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:129
 Composer\EventDispatcher\EventDispatcher->dispatchScript() at phar:///usr/local/bin/composer/src/Composer/Autoload/AutoloadGenerator.php:454
 Composer\Autoload\AutoloadGenerator->dump() at phar:///usr/local/bin/composer/src/Composer/Installer.php:352
 Composer\Installer->run() at phar:///usr/local/bin/composer/src/Composer/Command/InstallCommand.php:147
 Composer\Command\InstallCommand->execute() at phar:///usr/local/bin/composer/vendor/symfony/console/Command/Command.php:298
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:1040
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:301
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/composer/src/Composer/Console/Application.php:382
 Composer\Console\Application->doRun() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at phar:///usr/local/bin/composer/src/Composer/Console/Application.php:145
 Composer\Console\Application->run() at phar:///usr/local/bin/composer/bin/composer:88
 require() at /usr/local/bin/composer:29
```

Due to this line (154) in this file src/AutoloadGenerator.php : 
```php
$packageBaseDir = $filesystem->normalizePath($installationManager->getInstallPath($package));
```

This bug exist since 2.5.6 Composer version https://github.com/composer/composer/releases/tag/2.5.6 where getInstallPath return now null instead of empty string

Fix suggest for this line (154) in this file src/AutoloadGenerator.php :
```php
$packageBaseDir = $filesystem->normalizePath($installationManager->getInstallPath($package) ?? "");
```

**Comment by ivanvermeyen** [comment]
Hi,

Is this the same issue as #1?
This should have been fixed in 1.0.2:
https://github.com/codezero-be/composer-preload-files/commit/4bbbde895975e8c01fcda56acb029e522d76c7ce

**Comment by vbridier** [comment]
@ivanvermeyen mybad, after composer update, it's OK 

Thx for your time
